src/common/BaseAnnotator.ts

```diff
@@ -119,6 +119,9 @@ export default class BaseAnnotator extends EventEmitter {
         // Add classes to the parent elements to support CSS scoping
         this.annotatedEl.classList.add(CSS_LOADED_CLASS);
         this.containerEl.classList.add(CSS_CONTAINER_CLASS);
+        if (this.isFeatureEnabled('discoverability')) {
+            this.annotatedEl.classList.add('ba-discoverability-enabled');
```

Comment:
Should this be handled in the Preview SDK? If not, the other classes that are added are stored in constants and are also removed in `destroy`. Can we follow that same format?

Reply:
This seems reasonable since we're going to have to spread the discoverability logic to preview SDK anyway

Reply:
Sounds good I'll move it out of this PR.